lib/oli_web/templates/pow/session/user_new.html.eex

```diff
@@ -32,10 +32,6 @@
       <%= error_tag f, :password %>
     </div>
     <div class="mb-4 d-flex flex-row">
-      <div class="custom-control custom-checkbox flex-grow-1">
-        <%= checkbox f, :persistent_session, class: "custom-control-input" %>
```

Comment:
We should leave this checkbox here for users who may be accessing from a shared/non-secure browser and default to not setting a persistent session